project/Http4sPlugin.scala

```diff
@@ -334,4 +334,6 @@ object Http4sPlugin extends AutoPlugin {
   lazy val tomcatCatalina                   = "org.apache.tomcat"      %  "tomcat-catalina"           % "9.0.12"
   lazy val tomcatCoyote                     = "org.apache.tomcat"      %  "tomcat-coyote"             % tomcatCatalina.revision
   lazy val twirlApi                         = "com.typesafe.play"      %% "twirl-api"                 % "1.3.15"
+  lazy val scalafixCore                     = "ch.epfl.scala"          %% "scalafix-core"             % "0.9.0"
+  lazy val scalafixTestKit                  = "ch.epfl.scala"          % "scalafix-testkit"           % scalafixCore.revision % Test cross CrossVersion.full
```

Comment:
Nit: we usually keep these alphabetical.  It helps when maintaining across branches.